.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to Cryostat! 👋
+## Before contributing, make sure you have:
+* [ ] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
+* [ ] Rebased your branch on top the latest upstream `main` branch
+* [ ] Linked a relevant issue which this PR resolves
+* [ ] Linked any other relevant issues, PR's, or documentation (if applicable)
+* [ ] Attached at least one of the following labels is attached to the PR: [chore,ci,docs,feat,fix,test]
+* [ ] Request review from at least one of the following [reviewers](https://github.com/orgs/cryostatio/teams/reviewers)
```

Comment:
Not sure about this one. External contributors wouldn't know who to pick, so there probably isn't much value in making them pick or letting them pick. Maybe we can set this up to just automatically request review from the @cryostatio/reviewers group when someone who isn't a member of that group opens a PR?